modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java

```diff
@@ -152,11 +126,48 @@ public static void main(String[] args) {
         System.exit(hnd.execute(Arrays.asList(args)));
     }
 
+    /**
+     * @return prepared JULs logger.
+     */
+    private static Logger setupJavaLogger() {
+        Logger result;
+
+        result = Logger.getLogger(CommandHandler.class.getName() + "Log");
+        result.setLevel(Level.INFO);
+        result.setUseParentHandlers(false);
+
+        // Adding logging to file.
+        try {
+            String absPathPattern = new File(JavaLoggerFileHandler.logDirectory(U.defaultWorkDirectory()), "control-utility-%g.log").getAbsolutePath();
+
+            FileHandler fileHandler = new FileHandler(absPathPattern, 5 * 1024 * 1024, 5);
+            fileHandler.setFormatter(new JavaLoggerFormatter());
```

Comment:
Please add empty lines in accordance with codestyle.

Reply:
fixed